### PR TITLE
Add GradientElements collection

### PR DESCRIPTION
**Pull request by mgierlasinski**
Issues resolved: https://github.com/mgierlasinski/MagicGradients/issues/73

### CODE REVIEW

File: PlaygroundLite/PlaygroundLite.Android/Assets/AboutAssets.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Any raw assets you want to be deployed with your application can be placed in
```
Comment (bbenetskyy) [inline PlaygroundLite/PlaygroundLite.Android/Assets/AboutAssets.txt:1]: Do we really need this asset file?

File: PlaygroundLite/PlaygroundLite.Android/Resources/AboutResources.txt [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Images, layout descriptions, binary blobs and string dictionaries can be included
```
Comment (bbenetskyy) [inline PlaygroundLite/PlaygroundLite.Android/Resources/AboutResources.txt:1]: Here is the same I think we could remove that default file

File: PlaygroundLite/PlaygroundLite/App.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        public App()
+        {
+            InitializeComponent();
+
+            var page = FreshPageModelResolver.ResolvePageModel<MainViewModel>();
+            var basicNavContainer = new FreshNavigationContainer(page);
+            MainPage = basicNavContainer;
+
+            //MainPage = new MainView();
```
Comment (bbenetskyy) [inline PlaygroundLite/PlaygroundLite/App.xaml.cs:21]: This line is obsolete, we could remote as well(but it's not necessary to do it right now)

Comment (mgierlasinski) [reply PlaygroundLite/PlaygroundLite/App.xaml.cs:21]: Yup, that was quick testing of FreshMvvm.

File: PlaygroundLite/PlaygroundLite.Android/Assets/AboutAssets.txt (hunk 1 above)
Comment (mgierlasinski) [reply PlaygroundLite/PlaygroundLite.Android/Assets/AboutAssets.txt:1]: Nope, we don't really need it, it's File -> New project junk.

**Pull request merged**